### ISSUE TITLE
How can I convert the parameters with the suffixes "-contains", "-equals" to my model?

### ISSUE THREAD

**Issue by fernandofsan** [NonFactors/MVC5.Grid#141]
Hi everyone,
I'm recently using this library, but now i get confused how to bind the Filter params into my model.

**Comment by Muchiachio** [comment]
Automatic functionality is inside grid classes, it binds itself and filters automatically.
But if you want to do something by hand you will have to use raw query `.Request.Query`

**Comment by fernandofsan** [comment]
You mean Request.QueryString?

**Comment by Muchiachio** [comment]
Yes, mixed it up with asp net core version.

**Comment by fernandofsan** [comment]
Are there any samples in asp.net core using a model binder like: name-contains to an array of expressions and use this as a parameter of repository?

**Comment by Muchiachio** [comment]
Converting to expressions is the same in .core. As for model binding samples, I would use actual model binder [implementations](https://github.com/aspnet/Mvc/tree/d46948da1d4709bcb8ae0094074a307ece636b4d/src/Microsoft.AspNetCore.Mvc.Core/ModelBinding/Binders) in mvc repository as a reference. It has various binders like array/byte/complex and so on. I'm not sure what is a parameter of repository, why are you binding to expressions yourself though?

**Comment by Muchiachio** [comment]
But why do you need to parse them if grid filters and sorts for you?

**Comment by fernandofsan** [comment]
I think I'm reinventing a wheel. 
Let me try again with your tips.

**Comment by Muchiachio** [comment]
Did it work out for you?

**Comment by fernandofsan** [comment]
Yes, but i'm profiling the database and it seems the query is fetching all database.
Of course i'm passing the ToList to the grid, Is it possible to pass only an IQueryable?
The rest is working perfectly.

**Comment by fernandofsan** [comment]
Worked like a charm!
